Campaigns/apt sofacy.txt

```diff
@@ -0,0 +1,7 @@
+// Original Sigma Rule: https://github.com/Neo23x0/sigma/blob/master/rules/apt/apt_sofacy.yml
+// Questions via Twitter: @janvonkirchheim
+ProcessCreationEvents
+| where EventTime > ago(7d)
+| where ProcessCommandLine matches regex @'rundll32.exe %APPDATA%\.*\.dat\",.*'
```

Comment:
the `.*` at the end is redundant, because `matches regex` operator captures partial text

Reply:
Since `.` means "any character" in regex, you should change each `.` to `\.` meaning in regex real dot. Applies to the `.exe` in `rundll32.exe`